src/schema/annotation.yaml

```diff
@@ -192,6 +192,8 @@ classes:
       - subject
       - has function
     slot_usage:
+      id:
+        pattern: "KEGG\\.ORTHOLOGY:K\\d{5,}|EGGNOG:[a-zA-Z]+OG[a-zA-Z0-9]+|PFAM:PF\\d{5,}|TIGRFAM:TIGR\\d+|SUPFAM:\\w+|CATH:[1-6]\\.[0-9]+\\.[0-9]+\\.[0-9]+|PANTHER.FAMILY:PTHR\\d{5}(\\:SF\\d{1,3})?"
```

Comment:
this could match `fooKEGG.ORTHOLOGY:K12345blahblah`

I would go with a pattern

`^(REGEX1|REGEX2|...|REGEXn)$`

Reply:
THanks. I tried that, but it didn't work at [regexr](https://regexr.com/60e80)

Reply:
We retried the pattern with parens and anchors using [regexr's](https://regexr.com/60e80) test tab. Positive and negated negative tests all pass.